code/__DEFINES/maths.dm

```diff
@@ -84,9 +84,9 @@
 
 #define ISABOUTEQUAL(a, b, deviation) (deviation ? abs((a) - (b)) <= deviation : abs((a) - (b)) <= 0.1)
 
-#define ISEVEN(x) (x % 2 == 0)
+#define ISEVEN(x) (!((x) & 1))
```

Comment:
gonna admit i had a little laugh when i saw this since it's only used for roulette tables, but you think you can add a comment explaining why you're doing it this way so people don't have to use gitblame in the future to see why we don't just use the much simpler way it was before

Reply:
sure